drivers/bluetooth/hci/CMakeLists.txt

```diff
@@ -3,5 +3,7 @@
 zephyr_sources_ifdef(CONFIG_BT_H4       h4.c)
 zephyr_sources_ifdef(CONFIG_BT_H5       h5.c)
 zephyr_sources_ifdef(CONFIG_BT_SPI      spi.c)
-zephyr_sources_ifdef(CONFIG_BT_STM32_IPM      ipm_stm32wb.c)
+zephyr_sources_ifdef(CONFIG_BT_RPMSG	rpmsg.c)
+zephyr_sources_ifdef(CONFIG_BT_RPMSG_NRF53	rpmsg_nrf53.c)
+zephyr_sources_ifdef(CONFIG_BT_STM32_IPM	ipm_stm32wb.c)
```

Comment:
I suppose just indentation change in this line...

Reply:
yes, this is just indentation alignment